apps/web-giddh/src/app/ledger/ledger.component.html

```diff
@@ -1276,7 +1276,7 @@ <h3 class="modal-title bg" id="modal-title">Confirmation </h3>
                 <div class="modal-body pdL2 pdR2 clearfix" id="export-body">
                     <form name="newRole" novalidate class="" autocomplete="off">
                         <div class="modal_wrap mrB2">
-                            <h3>Do you want to delete the selected entries? </h3>
+                            <h3>Do you want to delete the selected entrie(s)? </h3>
```

Comment:
Should be entry(s)

Reply:
Please check the card and check the message mentioned there.